integration_test/gce/gce_testing.go

```diff
@@ -1500,57 +1500,66 @@ func InstallGsutilIfNeeded(ctx context.Context, logger *log.Logger, vm *VM) erro
 		return fmt.Errorf("this test does not know how to install gsutil on platform %q", vm.Platform)
 	}
 
-	// This is what's used on openSUSE.
-	repoSetupCmd := "sudo zypper --non-interactive refresh"
-
-	if strings.HasPrefix(vm.Platform, "sles-") {
-		// Use a vendored repo to reduce flakiness of the external repos.
-		// See http://go/sdi/releases/build-test-release/vendored for details.
-		repoArch := "x86-64"
-		if IsARM(vm.Platform) {
-			repoArch = "aarch64"
+	gcloudArch := "x86_64"
+	if IsARM(vm.Platform) {
+		gcloudArch = "arm"
+	}
+	gcloudPkg := "google-cloud-cli-453.0.0-linux-" + gcloudArch + ".tar.gz"
+	installFromTarball := `
+curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/` + gcloudPkg + `
+INSTALL_DIR="$(readlink --canonicalize .)"
+(
+	INSTALL_LOG="$(mktemp)"
+	# This command produces a lot of console spam, so we only display that
+	# output if there is a problem.
+	sudo tar -xf ` + gcloudPkg + ` -C ${INSTALL_DIR} 
+	sudo --preserve-env ${INSTALL_DIR}/google-cloud-sdk/install.sh -q &>"${INSTALL_LOG}" || \
+		EXIT_CODE=$?
+	if [[ "${EXIT_CODE-}" ]]; then
+		cat "${INSTALL_LOG}"
+		exit "${EXIT_CODE}"
+	fi
+)`
+	installCmd := `set -ex
+` + installFromTarball + `
+sudo ln -s ${INSTALL_DIR}/google-cloud-sdk/bin/gsutil /usr/bin/gsutil 
+`
+	// b/308962066: The GCloud CLI ARM Linux tarballs do not have bundled Python
+	// and the GCloud CLI requires Python >= 3.8. Install Python311 for ARM VMs
+	if IsARM(vm.Platform) {
+		// This is what's used on openSUSE.
+		repoSetupCmd := "sudo zypper --non-interactive refresh"
+		if strings.HasPrefix(vm.Platform, "sles-12") {
+			return fmt.Errorf("this test does not know how to install gsutil on platform %q", vm.Platform)
 		}
-		repo := "google-cloud-monitoring-sles12-" + repoArch + "-test-vendor"
+		// For SLES 15 ARM: use a vendored repo to reduce flakiness of the
+		// external repos. See http://go/sdi/releases/build-test-release/vendored
+		// for details.
 		if strings.HasPrefix(vm.Platform, "sles-15") {
-			repo = "google-cloud-monitoring-sles15-" + repoArch + "-test-vendor"
-		}
-		repoSetupCmd = `sudo zypper --non-interactive addrepo -g -t YUM https://us-yum.pkg.dev/projects/cloud-ops-agents-artifacts-dev/` + repo + ` test-vendor
+			repoSetupCmd = `sudo zypper --non-interactive addrepo -g -t YUM https://us-yum.pkg.dev/projects/cloud-ops-agents-artifacts-dev/google-cloud-monitoring-sles15-aarch64-test-vendor test-vendor
 sudo rpm --import https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 sudo zypper --non-interactive refresh test-vendor`
-	}
-
-	installCmd := `set -ex
+		}
 
+		installCmd = `set -ex
 ` + repoSetupCmd + `
-sudo zypper --non-interactive install --force-resolution --capability 'python>=3.6'
-sudo zypper --non-interactive install python3-certifi
+sudo zypper --non-interactive install python311 python3-certifi
```

Comment:
Why did we call the vendored package `python311` rather than just `python3`?

Reply:
for SLES 15 and OpenSUSE Leap, in the system repo, `Python3` only has version 3.6 available and `Python311` is an alternative installation. `yum upload` would keep the name when uploading to our repo. Is there anyway we can change the name of the downloaded RPM?